### PR TITLE
search working

### PR DESCRIPTION
**Pull request by Matthensley777**

### CODE REVIEW

**Comment by zoeames** [comment]
does this have a new feature?

**Comment by Matthensley777** [comment]
@zoeames I'm working on it, I'm trying to delete all contacts but it's not working yet

**Comment by morecallan** [comment]
👉 poke @Matthensley777

**Comment by Matthensley777** [comment]
@morecallan yes?

**Comment by morecallan** [comment]
Updates on this? Where are you at with your feature?

**Comment by Matthensley777** [comment]
@morecallan deleting from fire base but not the dom,

File: javascripts/services/ContactServices.js [hunk 1]
```diff
@@ -36,34 +36,55 @@ const buildNewContact = (Contacts, userId) => {
...
+        });
+    };
 
+    const deleteContact = (Id) => {
+        return $http.delete(`${FIREBASE_CONFIG.databaseURL}/Contacts/${Id}.json`);
+    };
 
 
+    const deleteAllContacts = () => {
+        return $http.delete(`${FIREBASE_CONFIG.databaseURL}/Contacts.json`);
```
Comment (morecallan) [inline javascripts/services/ContactServices.js]: This is deleting all the contacts (not just the ones that belong to that user).

**Review by morecallan — COMMENTED** [review]
hey @Matthensley777 - it looks like you are going to have to rethink this a little bit. You are going to need to find a way to delete all of the ones that have the correct uid tied to it.

**Comment by morecallan** [comment]
🐶

**Pull request merged**